examples/user_guide/03-Applying_Customization.ipynb

```diff
@@ -571,8 +565,7 @@
     "\n",
     "In addition to the formats described above for use by users, additional option formats are supported that are less user friendly for data exploration but may be more convenient for library authors building on HoloViews.\n",
     "\n",
-    "The first of these is the *`Option` list syntax* which is typically most useful outside of notebooks, a *literal syntax* that avoids the need to import `opts` and the finally a literal syntax that keeps *style* and *plot* options separate.\n",
-    "\n"
+    "The first of these is the *`Option` list syntax* which is typically most useful outside of notebooks, a *literal syntax* that avoids the need to import `opts`, and the finally a literal syntax that keeps *style* and *plot* options separate."
```

Comment:
```suggestion
     "The first of these is the *`Option` list syntax* which is typically most useful outside of notebooks, a *literal syntax* that avoids the need to import `opts`, and then finally a literal syntax that keeps *style* and *plot* options separate."
```

Reply:
Just FIY, only you can apply suggestions.